src/WPF/ArcGISRuntime.WPF.Viewer/Samples/GeometryEngine/DensifyAndGeneralize/DensifyAndGeneralize.xaml

```diff
@@ -17,21 +17,23 @@
                     <RowDefinition Height="Auto" />
                     <RowDefinition Height="Auto" />
                 </Grid.RowDefinitions>
-                <TextBlock Grid.Row="0"
-                           Text="Adjust the sliders to see the resulting generalized or densified polyline."
-                           TextWrapping="Wrap"
-                           Foreground="Red" />
-                <Label Grid.Row="1"
-                       Content="Max. Deviation (Generalize)" FontWeight="Bold" />
+                <TextBlock Text="Adjust the sliders to see the resulting generalized or densified polyline."
+                           Grid.Row="0" Margin="0,0,0,15"
+                           TextWrapping="Wrap" FontWeight="Bold" />
+                <TextBlock Text="Max. Deviation (Generalize)"
+                           Grid.Row="1"
+                           FontWeight="SemiBold" />
                 <Slider x:Name="DeviationSlider"
-                        Value="10" Minimum="1" Maximum="250"
-                        Grid.Row="2" Margin="5" />
-                <Label Grid.Row="3"
-                       Content="Max. Segment length (Densify)" FontWeight="Bold" />
+                        Grid.Row="2" Margin="0,5,0,0"
+                        Value="10" Minimum="1" Maximum="250" />
+                <TextBlock Text="Max. Segment length (Densify)"
+                           Grid.Row="3"
+                           FontWeight="SemiBold" />
                 <Slider x:Name="SegmentLengthSlider"
-                        Grid.Row="4" Margin="5"
+                        Grid.Row="4" Margin="0,5,0,5"
                         Value="100" Minimum="100" Maximum="500" />
-                <TextBlock x:Name="ResultLabel" Grid.Row="5"
+                <TextBlock x:Name="ResultLabel"
+                           Grid.Row="5"
                            Text="Adjust a slider to start"
                            TextWrapping="Wrap"
                            Foreground="Red" />
```

Comment:
I was worried it wouldn't stand out enough otherwise. It turns out blue text meets the need without being mangled so badly in the jpeg, so I'll update.